lib/internal/Magento/Framework/Mail/Template/TransportBuilder.php

```diff
@@ -394,7 +394,12 @@ protected function prepareMessage()
         }
 
         /** @var \Magento\Framework\Mail\MimePartInterface $mimePart */
-        $mimePart = $this->mimePartInterfaceFactory->create(['content' => $content]);
+        $mimePart = $this->mimePartInterfaceFactory->create(
+        	[
+        		'content' => $content,
+        		'type' => $part['type']
```

Comment:
I know the variable name is not coming from you but it looks like that $part is never defined as an array prior. Since it only uses type maybe it makes sense to rename `$part['type']` to `$partType`?

Reply:
I would agree. I suspect usage of the $part array is probably an artifact of an older version of the class. Replacing that with something simpler and less ambiguous would likely reduce confusion.